lab3/truncate.sql

```diff
@@ -0,0 +1,19 @@
+TRUNCATE TABLE subscription_daily_plans RESTART IDENTITY CASCADE;
+TRUNCATE TABLE daily_plan_meals RESTART IDENTITY CASCADE;
```

Comment:
To jest dodatkowe? Czy będziemy tego uzywać teraz zamiast drop.sql?

Reply:
Truncate czysci baze i resetuje indeksy, więc jest szybsze niż Drop -> Create: lepiej po prostu truncate jesli chcecie resetowac